core/newconfig/config.go

```diff
@@ -0,0 +1,100 @@
+package newconfig
```

Comment:
new?

Reply:
Just did it temporarily so that you can notice the changes easily.
Will replace it with the old config package once everything here is done